app/src/main/java/com/amaze/filemanager/ui/notifications/NotificationConstants.java

```diff
@@ -30,6 +30,7 @@
 
     public static final String CHANNEL_NORMAL_ID = "normalChannel";
     public static final String CHANNEL_FTP_ID = "ftpChannel";
+    public static final int ID_NOTIFICATION_WAIT =  9248;
```

Comment:
First, "NOTIFICATION" is redundant. Second, "WAIT_ID" already exists, why not use that?

Reply:
Will do, but change their name convention later. It's always more generalized to specific not the other way around. For eg. we use ````ID_WAIT```` not ````WAIT_ID````.